### ISSUE TITLE
He's dead, Jim!

### ISSUE THREAD

**Issue by ZanderBrown** [ravgeetdhillon/gnome-hackers#1]
![image](https://user-images.githubusercontent.com/17074021/74239295-7a2e0900-4ccf-11ea-83e7-af29222e5973.png)

The deploy step seems to be failing, and has been for sometime now

**Comment by ZanderBrown** [comment]
@ravgeetdhillon looks like you changed your password or something? it seems to work until it tries to push the updated site

**Comment by ravgeetdhillon** [comment]
I don't think that an issue. I have same script running on my other GitHub projects like [FantasyAI](https://gitHub.com/ravgeetdhillon/fantasyAI). I will work out on this next weekend.
